corehq/apps/hqadmin/service_checks.py

```diff
@@ -93,7 +94,7 @@ def check_elasticsearch():
     if cluster_health != 'green':
         return ServiceStatus(False, "Cluster health at %s" % cluster_health)
 
-    doc = {'_id': 'elasticsearch-service-check',
+    doc = {'_id': 'elasticsearch-service-check-{}'.format(random_hex[:7]),
            'date': datetime.datetime.now().isoformat()}
     send_to_elasticsearch('groups', doc)
```

Comment:
Should this be in a `try` block with the delete in the `finally` block? If an exception occurred in here the delete would not happen.

Reply:
good point. I'll follow up after today's deploy (wanted to get this in quickly).